homeassistant/components/vicare/icons.json

```diff
@@ -84,6 +84,15 @@
       },
       "compressor_phase": {
         "default": "mdi:information"
+      },
+      "ventilation_level": {
+        "default": "mdi:fan",
+        "state": {
+          "standby": "mdi:fan-off",
+          "levelone": "mdi:fan-speed-1",
+          "leveltwo": "mdi:fan-speed-2",
+          "levelthree": "mdi:fan-speed-3"
```

Comment:
Currently there is no `fan-speed-4` icon available, but I requested this: https://github.com/Templarian/MaterialDesign/issues/7578

Reply:
Will move this to the main fan entity.